build.cmd does not build solution on HP laptop (when Platform=MCD is pre-set)

I've got HP laptop with windows 7. When I run build.cmd, I've got the error "configuration is invalid"

    C:\Projects\dotnet\corefx>build.cmd
    C:\Projects\dotnet\corefx\src\System.Collections.Immutable.sln.metaproj : error MSB4126: указанная конфигурация решения "Release|MCD" недопустима. Укажите допустимую конфигурац
    ию решения с помощью свойств Configuration и Platform (например, MSBuild.exe Solution.sln /p:Configuration=Debug /p:Platform="Any CPU") или оставьте эти свойства пустыми, чтобы
    использовать конфигурацию решения по умолчанию. [C:\Projects\dotnet\corefx\src\System.Collections.Immutable.sln]

The reason of this error is that environment variable `Platform` is set to `MCD` on the laptop by default. I have to change build command in the build.cmd by adding `/p:Platform="Any CPU"` to compile the project.

Build script should check for allowed platforms and generate user-friendly error message, which says what to do to successfully compile the project when the platform is not supported.

--- a/build.cmd
+++ b/build.cmd
@@ -23,6 +23,11 @@ if not defined VisualStudioVersion (
 )
 
 :EnvSet
+:: Clear the 'Platform' env variable for this session,
+:: as it's a per-project setting within the build, and
+:: misleading value (such as 'MCD' in HP PCs) may lead
+:: to build breakage (issue: #69).
+set Platform=
 
 :: Log build command line
 set _buildproj=%~dp0build.proj

